install.sh

```diff
@@ -90,73 +87,63 @@ function curl () {
   # the 'curl: (7) Couldn't connect to server' error, a for loop is used
   # here.
   for (( i = 0; i < 4; i++ )); do
-    # force TLS as we know GitLab supports it
-    env -u LD_LIBRARY_PATH ${CURL} --ssl-reqd --tlsv1.2 -C - "${@}" && return 0
-    echo_info "Retrying, $((3-i)) retries left."
+    # Force TLS as we know GitLab supports it
+    if [ "$(env -u LD_LIBRARY_PATH "${CURL}" --ssl-reqd --tlsv1.2 -C - "${@}")" ]; then
+      echo_info "Retrying, $((3-i)) retries left."
+    else
+      return 0
+    fi
```

Comment:
I don't think this logic will work as expected.  If the condition is true, you will want to return 0, correct?

Reply:
Fixed.

Reply:
*Broken, one moment/

Reply:
We are testing this script on a powerwashed system, correct?

Reply:
On a docker image.

Reply:
At any rate, it's the same logic that's currently in the install script: https://github.com/chromebrew/chromebrew/blob/570cc4b8507a162f6f4f7cd2da7b3345e89a6058/install.sh#L94

Reply:
This doesn't look like the same logic.  If the `env ...` line returns true, the loop should exit.